Sources/Apodini/Semantic Model Builder/SharedSemanticModelBuilder.swift

```diff
@@ -97,15 +98,16 @@ class SharedSemanticModelBuilder: SemanticModelBuilder, InterfaceExporterVisitor
             parameters: parameterBuilder.parameters
         )
         
-        webService.addEndpoint(&endpoint, at: paths)
+        webService.addEndpoint(endpoint, at: paths)
     }
     
     override func finishedRegistration() {
         super.finishedRegistration()
         
         webService.finishedParsing = true
         
-        webService.root.printTree() // currently only for debugging purposes
+        // currently only for debugging purposes
+        print(webService.root.debugDescription)
```

Comment:
Don't forget to remove debugging `prints` before this PR gets merged 🙃

Reply:
You are right 🙂